File: javascript/EliminacionPrimerNumero.js
let numeros = [11, 21, 31, 41, 51];
console.log("Asi se ve la lista antes de la eliminacion");
console.log(numeros.toString());

let nuevo = []
for (let j = 0; j<numeros.length -1; j++ ){
    nuevo[j] = numeros[j + 1];
}
console.log("Despues de la eliminacion queda: ");
console.log(nuevo.toString());

File: cSharp/EliminacionPrimerNumero.cs

using System;

class program
{
    static void Main(string[] args)
    {
        int [] numeros = { 11, 21, 31, 41,51 };
        Console.WriteLine("Antes de la eliminacion, el arreglo es: ");
        Console.WriteLine(string.Join(", ", numeros));

        int [] nuevo = new int[numeros.Length - 1];

        for (int j = 0; j < numeros.Length - 1; j++)
        {
            nuevo[j] = numeros[j + 1];
        }

        Console.WriteLine("Despues de la eliminacion, el arreglo es: ");
        Console.WriteLine(string.Join(", ", nuevo));
    }
}